Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "era_test_node"
-version = "0.1.0-alpha.14"
+version = "0.1.0-alpha.15"
```

Let's update this to next version so the binary has the correct release (this is feedback per a specific partner, and we can go back to normally scheduled programming after this). This will be the last commit/PR for `0.1.0-alpha.16`
```suggestion
version = "0.1.0-alpha.16"
```